jwql/website/apps/jwql/admin.py

```diff
@@ -43,5 +41,15 @@ class ObservationAdmin(admin.ModelAdmin):
 
 @admin.register(RootFileInfo)
 class RootFileInfoAdmin(admin.ModelAdmin):
-    list_display = ('root_name', 'obsnum', 'proposal', 'instrument', 'viewed')
+    list_display = ('root_name', 'obsnum', 'proposal', 'instrument', 'viewed', 'filter', 'aperture', 'detector', 'read_patt_num', 'read_patt', 'grating', 'subarray', 'pupil', 'exp_type')
     list_filter = ('viewed', 'instrument', 'proposal', 'obsnum')
+
+
+@admin.register(Anomalies)
+class AnomaliesAdmin(admin.ModelAdmin):
+    list_display = ('root_file_info', 'cosmic_ray_shower', 'diffraction_spike', 'excessive_saturation', 'guidestar_failure', 'persistence', 'crosstalk', 'data_transfer_error',
```

Comment:
I was going to suggest getting these from constants.py, although I guess we may not always have all anomalies listed in constants.py in the models. So in that respect it's better to keep this separate from constants.py?

Reply:
also!  I'm still running a final test, I think the update will take many hours on my machine, and it will re-start if I lose any connectivity (it has happened once already), so there may be a small change or two that will come in if I find anything else
If you prefer I can tag you in a comment when I'm confident all is good